app/uk/gov/hmrc/fhregistrationfrontend/controllers/BusinessPartnerCorporateBodyTradingNameController.scala

```diff
@@ -34,12 +34,16 @@ class BusinessPartnerCorporateBodyTradingNameController @Inject()(
 
   import actions._
 
+  val BusinessType = "corporateBody"
```

Comment:
naming convention - please use businessType instead of BusinessType as its not a class